presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueryFramework.java

```diff
@@ -490,6 +495,12 @@ protected QueryRunner getQueryRunner()
         return queryRunner;
     }
 
+    protected QueryRunner getExpectedQueryRunner()
+    {
+        checkState(expectedQueryRunner != null, "expectedQueryRunner not set");
+        return (QueryRunner) expectedQueryRunner;
```

Comment:
Is this conversion valid?  `QueryRunner` extends the `ExpectedQueryRunner`. Wonder how this works.

Reply:
From an API standpoint, ExpectedQueryRunner only has `MaterializedResult execute(Session session, @Language("SQL") String sql, List<? extends Type> resultTypes);`. Can we use this function or add new execute API to the ExpectedQueryRunner class?

Reply:
Thanks for the suggestion. I have updated the code and removed `getExpectedQueryRunner` method.